test/generators/PaymentGenerator.scala

```diff
@@ -54,7 +54,7 @@ trait PaymentGenerator extends BaseGenerator with AmlsReferenceNumberGenerator {
     amlsRefNo,
     safeId,
     ref,
-    desc,
+    Some(desc),
```

Comment:
so payment generator will always generate some description? what about no-description?

Reply:
@slavvco - As this code is on the old API there will always be a description and this is just a test generator. The next set of PRs to cover the API endpoint change will add some tests that cover no description. Make sense?